### ISSUE TITLE
Update dependencies to versions under maintenance.

### ISSUE THREAD

**Issue by dpordomingo** [ericwooley/thewooleyway#16]
The latest version of `@tww/terminus-maximus` depends on `npm@4.6.1` who depends on `hawk@3.1.3` which is pretty outdated, and no longer under maintenance.

```shell
$ npm ls hawk
@tww/terminus-maximus@2.0.6
  └─┬ npm@4.6.1
    └─┬ request@2.81.0
      └── hawk@3.1.3 
```

so installing `@tww/terminus-maximus` causes many warnings:
```shell
warning @tww/terminus-maximus > npm > request > hawk > hoek@2.16.3: This version is no longer maintained. Please upgrade to the latest version.
warning @tww/terminus-maximus > npm > request > hawk > boom@2.10.1: This version is no longer maintained. Please upgrade to the latest version.
warning @tww/terminus-maximus > npm > request > hawk > boom > hoek@2.16.3: This version is no longer maintained. Please upgrade to the latest version.
warning @tww/terminus-maximus > npm > request > hawk > cryptiles@2.0.5: This version is no longer maintained. Please upgrade to the latest version.
warning @tww/terminus-maximus > npm > request > hawk > cryptiles > boom@2.10.1: This version is no longer maintained. Please upgrade to the latest version.
warning @tww/terminus-maximus > npm > request > hawk > sntp > hoek@2.16.3: This version is no longer maintained. Please upgrade to the latest version.
```

Is there a way to update `@tww/terminus-maximus` to rely on newer libs?

**Comment by ericwooley** [comment]
@dpordomingo sure, I can probably get to it this weekend, or a PR would be accepted. 

I'm not sure if npm needs to be explicitly depended on, it will probably work fine if npm is removed as a dependency.

**Comment by ericwooley** [comment]
@dpordomingo Sorry this took a while, i don't really use this lib anymore, so it kept getting pushed down in priority.

I just published 2.0.7 which removes npm as a dependency alltogether, since it's not really needed.

Let me know if that works for you!
